client/src/components/CalendarPage.js

```diff
@@ -35,6 +35,7 @@ class CalendarPage extends React.Component {
     API.getLectures(startOfWeek,endOfWeek,this.props.authUser.role,this.props.authUser.id)
     .then((res)=>{
       console.log("RES"+res[0].course_id)
+      res[3].deleted_at = "2020-11-19 08:30:00.000+01:00";
```

Comment:
Should this be here?

Reply:
I think that all the various console.log and hardcoded lines should be removed before merging.

Reply:
Whoops, was some test case.. I've then stopped working on that because I knew that that Component was a bit different in another branch. But yes, definitelly to remove that